src/main/java/checklst/task/Event.java

```diff
@@ -0,0 +1,58 @@
+package checklst.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import checklst.exception.ChecklstException;
+
+/**
+ * The Event class represents a Task which has a specific DateTime.
+ */
+public class Event extends Task {
+
+    protected final LocalDate dateTime;
```

Comment:
I would prefer to use `date` instead of `dateTime`, as dateTime suggests that the information also includes a time element

Reply:
Thanks for the suggestion! Changed accordingly.